### ISSUE TITLE
AnyPackage v0.1.1 failing to load due to Immutable library v7 on Windows GitHub runners

### ISSUE THREAD

**Issue by ethanbergstrom** [AnyPackage/AnyPackage#16]
Installing AnyPackage v0.1.1 from PSGallery on a Server 2022 GitHub runner and importing it yields the below error. This issue was not seen with v0.1.0.

The issue is:

- **also seen** on Github's Server 2019 runner, with both PowerShell 5.1 **and** PowerShell 7
- **not** seen when installing from scratch in a Windows 11 Sandbox **or** on a Server 2022 instance spun up in OCI **or** on Mac OS Ventura

```
Error: [-] Discovery in D:\a\AnyPackage.Chocolatey\AnyPackage.Chocolatey\test\AnyPackage.Chocolatey.Static.Tests.ps1 failed with:
Message
  System.IO.FileLoadException: Could not load file or assembly 'System.Collections.Immutable, Version=7.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
  at System.Management.Automation.Runspaces.InitialSessionState.Bind_LoadAssemblies(ExecutionContext context)
  at System.Management.Automation.Runspaces.InitialSessionState.Bind(ExecutionContext context, Boolean updateOnly, PSModuleInfo module, Boolean noClobber, Boolean local, Boolean setLocation)
  at Microsoft.PowerShell.Commands.ModuleCmdletBase.LoadModuleManifest(String moduleManifestPath, ExternalScriptInfo manifestScriptInfo, Hashtable data, Hashtable localizedData, ManifestProcessingFlags manifestProcessingFlags, Version minimumVersion, Version maximumVersion, Version requiredVersion, Nullable`1 requiredModuleGuid, ImportModuleOptions& options, Boolean& containedErrors)
  at <ScriptBlock>, D:\a\AnyPackage.Chocolatey\AnyPackage.Chocolatey\test\AnyPackage.Chocolatey.Static.Tests.ps1: line 1
  at <ScriptBlock>, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 2995
  at Invoke-File, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 3004
  at Invoke-BlockContainer, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 2927
  at Discover-Test, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 1464
  at Invoke-Test, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 2437
  at Invoke-Pester<End>, C:\Program Files\WindowsPowerShell\Modules\Pester\5.4.0\Pester.psm1: line 5272
  at <ScriptBlock>, D:\a\_temp\9354fe32-[15](https://github.com/AnyPackage/AnyPackage.Chocolatey/actions/runs/4092546065/jobs/7057362407#step:6:16)81-46e4-b7cb-982bfd5a7088.ps1: line 2
  at <ScriptBlock>, <No file>: line 1
```

**Comment by ThomasNieto** [comment]
Do you get the same error on an earlier windows version?

**Comment by ethanbergstrom** [comment]
This is what I see when updating the module on my Windows 11 workstation:
```
Install-Package: The version '0.1.1' of the module 'AnyPackage' being installed is not catalog signed. Ensure that the version '0.1.1'
of the module 'AnyPackage' has the catalog file 'AnyPackage.cat' and signed with the same publisher 'CN=Thomas Nieto,
O=Thomas Nieto, L=Overland Park, S=Kansas, C=US' as the previously-installed module 'AnyPackage' with version '0.1.0'
under the directory 'C:\Users\ethan\OneDrive\Documents\PowerShell\Modules\AnyPackage\0.1.0'. If you still want to
install or update, use -SkipPublisherCheck parameter.
```

**Comment by ThomasNieto** [comment]
Try the -SkipPublisherCheck or uninstall the module first. I have to get the new certificate in place to sign. In the meantime that version isn't signed.

**Comment by ThomasNieto** [comment]
I'll go ahead and revert it and publish a new version until we figure out the issue. There isn't a need for AnyPackage to use the latest version.

**Comment by ThomasNieto** [comment]
The interesting thing while looking at the Chocolatey tests is that looks like your manual DLL import test works.

```
[reflection.assembly]::LoadFile($(Get-ChildItem 'C:\Users\runneradmin\Documents\PowerShell\Modules\' -Recurse -Filter 'System.Collections.Immutable.dll' | Select-Object -ExpandProperty FullName)) | Format-List

CodeBase               : file:///C:/Users/runneradmin/Documents/PowerShell/Modules/AnyPackage/0.1.1/System.Collections.
                         Immutable.dll
EntryPoint             : 
EscapedCodeBase        : file:///C:/Users/runneradmin/Documents/PowerShell/Modules/AnyPackage/0.1.1/System.Collections.
                         Immutable.dll
FullName               : System.Collections.Immutable, Version=[7](https://github.com/AnyPackage/AnyPackage.Chocolatey/actions/runs/4093022474/jobs/7058166902#step:6:8).0.0.0, Culture=neutral, 
                         PublicKeyToken=b03f5f7f11d50a3a
GlobalAssemblyCache    : False
HostContext            : 0
ImageFileMachine       : 
ImageRuntimeVersion    : v4.0.3031[9](https://github.com/AnyPackage/AnyPackage.Chocolatey/actions/runs/4093022474/jobs/7058166902#step:6:10)
Location               : C:\Users\runneradmin\Documents\PowerShell\Modules\AnyPackage\0.1.1\System.Collections.Immutable.dll
ManifestModule         : System.Collections.Immutable.dll
MetadataToken          : 
PortableExecutableKind : 
ReflectionOnly         : False
```

**Comment by ThomasNieto** [comment]
Published v0.1.2, go ahead and try your CI build.

**Comment by ethanbergstrom** [comment]
Yup, that got me past it. Thanks!
